src/index.html

```diff
@@ -11,7 +11,177 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <h1></h1>
     <script type="text/javascript" src="./main.js"></script>
+
+  <form 
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+    method="post">
+    <fieldset class="first_info">
+      <legend>Personal information</legend>
+        <label class="form-fild">
+          Surname:
+
+          <input 
+            type="text" 
+            name="Surname"
+            required
+            autocomplete="off">
+        </label>
+
+        <label class="form-fild">
+          Name:
+
+          <input 
+            type="text" 
+            name="Name"
+            autocomplete="off">
```

Comment:
```suggestion
          <input 
            type="text" 
            name="Name"
            autocomplete="off"
          >
```
fix it in other places